docs/tidb_http_api.md

```diff
@@ -251,6 +251,67 @@
     }
     ```
 
+    *Hint: The method to convert the Hex format key returned by TiDB API into the format recognized by [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control).*
+    
+    step 1: Get the hex of the start key from tidb.
+     ```shell
+        $curl http://127.0.0.1:10080/mvcc/key/test/t1/1
+        {
+        "key": "7480000000000008C65F728000000000000001",
+        "region_id": 10,
+        "value": {
+            "info": {
+                "writes": [
+                    {
+                        "start_ts": 445971968923271174,
+                        "commit_ts": 445971968923271175,
+                        "short_value": "gAACAAAAAgMIAAkAc2hpcmx5YTQi"
+                    },
+                    {
+                        "start_ts": 445971959499980803,
+                        "commit_ts": 445971959499980804,
+                        "short_value": "gAACAAAAAgMIAAkAc2hpcmx5YTQL"
+                    }
+                ]
+            }
+        }
+        }
+     ```
+     step 2: convert the hex key from tidb to escaped key with [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control)
+     ```shell
+      ./tikv-ctl --to-escaped '7480000000000008C65F728000000000000001'
+      t\200\000\000\000\000\000\010\306_r\200\000\000\000\000\000\000\001
+     ```
+     step 3: Encode the key to make it memcomparable in tikv with [tikv-ctl](https://docs.pingcap.com/tidb/stable/tikv-control)
+     ```
+     ./tikv-ctl --encode 't\200\000\000\000\000\000\010\306_r\200\000\000\000\000\000\000\001'
+     7480000000000008FFC65F728000000000FF0000010000000000FA
+     ```
+     step 4: convert the hex key to escaped key since `tikv-ctl scan` only accepts escaped key.
```

Comment:
```suggestion
     step 4: convert the key to escaped format again since `tikv-ctl scan` only accepts escaped key, but the `--encode` command outputs the key in hex format.
```